File: commons/src/main/java/com/ddubson/CommandLineLogAdapter.java
package com.ddubson;

public class CommandLineLogAdapter implements LogAdapter {
    @Override
    public void info(String message) {
        System.out.println(message);
    }

    @Override
    public void info(String message, ANSIColor color) {
        System.out.format("%s %s %s\n", color.color(), message, ANSIColor.ANSI_RESET.color());
    }
}

File: commons/src/main/java/com/ddubson/CommandLineLogAdapter.kt
package com.ddubson

class CommandLineLogAdapter : LogAdapter {
    override fun info(message: String) {
        println(message)
    }

    override fun info(message: String, color: ANSIColor) {
        String.format("%s %s %s\n", color.color(), message, ANSIColor.ANSI_RESET.color())
    }

}